heroku/templates/src/main/java/package/config/_HerokuDatabaseConfiguration.java

```diff
@@ -32,32 +32,22 @@ public void setEnvironment(Environment environment) {
     public DataSource dataSource(<% if (hibernateCache == 'hazelcast') { %>CacheManager cacheManager<% } %>) {
         log.debug("Configuring Heroku Datasource");
 
-        String herokuUrl = propertyResolver.getProperty("heroku-url");
+        String herokuUrl = System.getenv("JDBC_DATABASE_URL");
```

Comment:
I like the `getenv` because it allows for "zero config" where it just works on Heroku. But I don't like it enough to argue in favor of it. I'll update with the `propertyResolver.getProperty("url")` and `--spring.datasource.url=$JDBC_DATABASE_URL` in the Procfile